src/main/kotlin/racingCar/ui/Input.kt

```diff
@@ -0,0 +1,13 @@
+package racingCar.ui
+
+class Input {
```

Comment:
```suggestion
object Input {
```
무언가를 캡슐화하고 있지 않다면, object를 활용해도 좋을 것 같아요. (이하 관련 내용 동일)